.github/workflows/build.yml

```diff
@@ -14,87 +15,65 @@ jobs:
         with:
           access_token: ${{ github.token }}
   build:
-    name: Release ${{ matrix.os }}
+    name: Release
     strategy:
       fail-fast: false
-      matrix:
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
-    runs-on: ${{ matrix.os }}
+    runs-on: macos-latest
+    env:
+      RUST_BACKTRACE: "1"
+      SCCACHE_GHA_ENABLED: "true"
+      RUSTC_WRAPPER: "sccache"
     steps:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.ref }}
-      - uses: actions-rs/toolchain@v1
+      - name: setup sccache
+        uses: mozilla-actions/sccache-action@v0.0.3
         with:
-          toolchain: stable
-      - uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --release
-      - name: Upload Mac / Linux Binaries
-        if: ${{ matrix.os != 'windows-latest' }} 
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.os }}
-          path: target/release/rewatch
-          retention-days: 1
-      - name: Upload Windows Binaries
-        if: ${{ matrix.os == 'windows-latest' }} 
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.os }}
-          path: target/release/rewatch.exe
-          retention-days: 1
-      - name: Attach Mac / Linux Binaries to release
-        if: ${{ matrix.os != 'windows-latest' }} 
-        uses: svenstaro/upload-release-action@v2
+          version: "v0.4.0"
+      - uses: dtolnay/rust-toolchain@master
         with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: target/release/rewatch
-          asset_name: rewatch-${{ matrix.os }}
-          tag: ${{ github.ref }}
-          overwrite: true
-      - name: Attach Windows Binaries to release
-        if: ${{ matrix.os == 'windows-latest' }} 
-        uses: svenstaro/upload-release-action@v2
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: target/release/rewatch.exe
-          asset_name: rewatch-${{ matrix.os }}
-          tag: ${{ github.ref }}
-          overwrite: true
-
-  npm-release:
-    name: NPM Release
-    runs-on: ubuntu-latest
-    if: github.event.action == 'published'
-    needs: 
-      - build
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+          toolchain: stable
+      - name: Install zig
```

Comment:
Oh nice! 😎

Reply:
apparently zig is very good at cross compiling! (and rust doesn't have this natively)